.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 temp/
+.idea
```

Comment:
Isn't .idea a folder at the project root? In that case, it should be written as:

```
/.idea/
```

I like to try to make exclusions as specific as possible to avoid unintended consequences.